src/extension.ts

```diff
@@ -29,23 +30,25 @@ export async function activate(context: ExtensionContext): Promise<void> {
     workspace.registerTextDocumentContentProvider(PrettyAnsiContentProvider.scheme, prettyAnsiContentProvider)
   );
 
-  const showPretty = async (options?: TextDocumentShowOptions) => {
-    const actualUri = window.activeTextEditor?.document.uri;
+  const showPretty = async (forcePreview: boolean = false, options: TextDocumentShowOptions = { viewColumn: ViewColumn.Active}) => {
+    if (forcePreview || workspace.getConfiguration(`${extensionId}`).autoPreview) {
+      const editor = window.activeTextEditor
+      if (editor && editor.document && editor.document.languageId === 'ansi' && editor.document.uri.scheme === 'file') {
```

Comment:
I'd change the logic from "only `file` scheme" to "every non-ours scheme". This will make it integrate better with other extensions, e.g. with the "view file at revision" feature of GitLens.